temporalcli/commands.workflow.go

```diff
@@ -200,16 +200,17 @@ func (c *TemporalWorkflowUpdateCommand) run(cctx *CommandContext, args []string)
 		return err
 	}
 
-	request := &client.UpdateWorkflowWithOptionsRequest{
+	request := client.UpdateWorkflowOptions{
 		WorkflowID:          c.WorkflowId,
 		RunID:               c.RunId,
 		UpdateName:          c.Name,
 		UpdateID:            c.UpdateId,
 		FirstExecutionRunID: c.FirstExecutionRunId,
 		Args:                input,
+		WaitForStage:        client.WorkflowUpdateStageCompleted,
```

Comment:
Not sure we should specify a default here for the same reason we have a default in the SDK

Reply:
I was assuming that a different update PR would decide what to expose. I could open an issue so that we don't forget...